src/app/components/harvest/widgets/validations.component.html

```diff
@@ -0,0 +1,99 @@
+<!-- TODO Show something when not meta review stage -->
+<ng-container *ngIf="isMetaReviewStage$ | async">
+  <h5>Instructions</h5>
+
+  <hr />
+
+  <p>
+    Open up some rows by clicking on the folder icon, or path. As you expose
+    issues, we will provide more customised help
```

Comment:
Don't mention rows - they're just folder.
Add a tip icon?

```suggestion
    You can open and close folders by clicking on them.
```